codepen-data.js

```diff
@@ -40,27 +42,36 @@ var types = {
 				}
 			}
 		});
+
+		// If there are no scripts the should at least be HTML
+		if(!result) {
+			result = {
+				html: HTML,
+				editors: DEFAULT_EDITORS
```

Comment:
I think we might want to only expand the `HTML` editor in this case.  It might be `1000`?